ext/release-lead.rake

```diff
@@ -78,7 +78,7 @@ namespace :release_lead do
           result[name] = 'No URL provided'
           next
         end
-        name = url_to_component_name(url)
+        name = url_to_component_name(url) == 'facter' ? json_name_to_component_name(component) : url_to_component_name(url)
```

Comment:
This is a bit hard to read, as you evaluate `url_to_component_name(url)` and then pass it to the short `if` and assign it back to `name`. I would suggest doing it one step at a time, like:
```ruby
name = url_to_component_name(URL)
name = json_name_to_component_name(component) if name == 'facter'
```